src/main/java/seedu/address/model/pet/UniquePetList.java

```diff
@@ -33,7 +33,7 @@
      */
     public boolean contains(Pet toCheck) {
         requireNonNull(toCheck);
-        return internalList.stream().anyMatch(toCheck::isSamePet);
```

Comment:
Maybe can consider changing the equals in `isSamePet` to be case insensitive!
I think `isSamePet` is defined in Pet.java!

Reply:
I have already changed the `equals()` method of `Name.java` such that it is case insensitive, hence `isSamePet()` should also be case insensitive.